### PR TITLE
Info endpoint 312

### PR DESCRIPTION
**Pull request by Th3nn3ss**

### CODE REVIEW

File: backend/channel_plugin/channel_plugin/info/views.py.a745cf90f9bf5c67af23ce3950aca471.tmp [hunk 1]
```diff
@@ -0,0 +1,54 @@
+from django.conf import settings
```
Comment (charles-co) [inline backend/channel_plugin/channel_plugin/info/views.py.a745cf90f9bf5c67af23ce3950aca471.tmp:1]: what's this file? please remove

Comment (Th3nn3ss) [reply backend/channel_plugin/channel_plugin/info/views.py.a745cf90f9bf5c67af23ce3950aca471.tmp]: Done

**Comment by charles-co** [comment]
ready for PR review by mark

**Comment by Th3nn3ss** [comment]
slack name: dennis_d_menace
live link: https://channels.zuri.chat/api/

**Pull request merged**